### PR TITLE
Card variants fixes and additional card variants

### PR DESCRIPTION
**Pull request by Kevin-Solomon**
## New card Variant
image

## Add new animation when the card hovered
[Card CodeSandBox Link](https://qg4h2.csb.app/pages/card/card.html)

### CODE REVIEW

File: pages/card/card.css [hunk 1]
```diff
@@ -11,6 +13,21 @@
   border-radius: 5px;
   border: 1px solid #2563eb;
   box-shadow: rgba(0, 0, 0, 0.35) 0px 5px 15px;
+  transition: transform 0.3s ease-in-out;
+}
+.product-card-badge {
+  position: absolute;
+  background-color: #e67a00;
+  /* width: 4rem; */
```
Comment (gadgetfather) [inline pages/card/card.css:21]: You can remove this comment before merging

File: pages/card/card.html [hunk 2]
```diff
@@ -4,27 +4,37 @@
...
+      <i class="far fa-heart fa-lg"></i>
+    </div>
+    <h4>Real Madrid Jersey</h4>
+    <h4>₹2000</h4>
+    <button class="btn">Add to cart</button>
+  </div>
+  <div class="product-card">
+    <div class="card-image-container">
+      <img class="responsive-img" src="../../assets/real-madrid-21-22-home-jersey.jpg">
       <i class="far fa-heart fa-lg"></i>
```
Comment (gadgetfather) [inline pages/card/card.html:25]: Also while hovering on this icon you can make its color red

**Review by gadgetfather — APPROVED** [review]
Clean code. PR is good to merge

**Review by kart-c — APPROVED** [review]

File: pages/card/card.css [hunk 3]
```diff
@@ -11,6 +13,21 @@
   border-radius: 5px;
   border: 1px solid #2563eb;
```
Comment (ruchiragoswami) [inline pages/card/card.css:14]: Just want to draw your attention, the color is hard-coded. Not sure if you want it that way or in a variable.

**Review by ruchiragoswami — APPROVED** [review]
Great work Kevin ! 
Just a general comment, colors are hard-coded at some places. Not sure if you want to keep it that way or in a variable.

File: pages/card/card.html (hunk 2 above)
Comment (Kevin-Solomon) [reply pages/card/card.html:25]: Yes but only the border color of the heart changes to red.
To make the actual heart fill with red we need to add a new class name.

File: pages/card/card.css (hunk 3 above)
Comment (Kevin-Solomon) [reply pages/card/card.css:14]: This is just a temporary color. 
I will change this once i get approval on the CL to use its primary color variable.

**Pull request merged**